onnxscript/tests/function_libs/torch_lib/ops_test_data.py

```diff
@@ -506,35 +506,29 @@ def _where_input_wrangler(
     TorchLibOpInfo("baddbmm", core_ops.aten_baddbmm),
     TorchLibOpInfo("bmm", core_ops.aten_bmm),
     TorchLibOpInfo("broadcast_to", core_ops.aten_broadcast_to),
-    TorchLibOpInfo(
-        "cat",
-        core_ops.aten_cat,
-        input_wrangler=_cat_input_wrangler,
-    ).skip(
+    TorchLibOpInfo("cat", core_ops.aten_cat, input_wrangler=_cat_input_wrangler).skip(
         matcher=lambda sample: sample.input[0].equal(torch.tensor([])),
         reason="cat does not support zero-dim tensors yet",
     ),
     TorchLibOpInfo("ceil", core_ops.aten_ceil),
-    TorchLibOpInfo(
-        "chunk",
-        core_ops.aten_chunk,
-    ).xfail(
+    TorchLibOpInfo("chunk", core_ops.aten_chunk).xfail(
         dtypes=[torch.float16],
         reason="fixme: SplitToSequence op inference failed. https://github.com/microsoft/onnxruntime/issues/16006",
     ),
     TorchLibOpInfo("clamp_max", core_ops.aten_clamp_max),
     TorchLibOpInfo("clamp_min", core_ops.aten_clamp_min),
     TorchLibOpInfo("clone", core_ops.aten_clone),
+    TorchLibOpInfo("concat", core_ops.aten_concat).skip(reason="fixme: ORT aborts"),
+    TorchLibOpInfo("concatenate", core_ops.aten_concatenate).skip(reason="fixme: ORT aborts"),
```

Comment:
Is there an issue in ORT repo for tracing?

Reply:
Added https://github.com/microsoft/onnxruntime/issues/16619